<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>679</width>
    <height>409</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineAddName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAddSurname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>127</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineAddAge">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="SexBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="LevelBox">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>30</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>30</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Dodaj</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineFindName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>120</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="FindButton">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>120</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Znajdź</string>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteButton">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>93</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDeleteName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>200</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDeleteSurname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>200</y>
      <width>127</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineChangeName">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>280</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineChangeSurname">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>280</y>
      <width>127</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineChangeAge">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>280</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="PrintButton">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>343</y>
      <width>341</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Wyświetl oraz zapisz listę</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>11</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Imię</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>10</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wiek</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>94</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Imię lub Nazwisko</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>177</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Imię</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>180</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wiek</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>260</y>
      <width>29</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Imię</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>144</x>
      <y>260</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Nazwisko</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>260</y>
      <width>33</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Wiek</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineDeleteAge">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>200</y>
      <width>126</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="ChangeLevelBox">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>280</y>
      <width>75</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="ChangeButton">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>280</y>
      <width>106</width>
      <height>29</height>
     </rect>
    </property>
    <property name="text">
     <string>Zmień poziom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>10</y>
      <width>27</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Płeć</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Poziom</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>411</x>
      <y>260</y>
      <width>49</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Poziom</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>679</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
